Sources/MeiliSearch/Model/SearchParameters.swift

```diff
@@ -40,22 +40,22 @@ public struct SearchParameters: Codable, Equatable {
     public let facetsDistribution: [String]?
 
     /// Whether to return the raw matches or not.
-    public let matches: Bool
+    public let matches: Bool?
 
     // MARK: Initializers
 
     init(
-        query: String,
-        offset: Int = Default.offset.rawValue,
-        limit: Int = Default.limit.rawValue,
+        query: String?,
```

Comment:
This init is not needed anymore because structs with nil parameters have an implicit init ready for the values.

Reply:
Not sure to understand. What would you like to be changed? 🙂

Reply:
Forget about it, you are correct.